A11y microsite (#537)
hms-dbmi/gehlenborglab-website: _teams/accessibility/our-accessibility-journey.md

---
# make sure to add page to _data/teams.yml

name: Our Accessibility Journey
team_id: accessibility
active: true
---

<!-- Every web interface should be accessible, but inaccessible web experiences are the norm. -->

When the HIDIVE lab began our accessibility research, we first focused on improving screen reader accessibility for data visualizations. 
Recognizing that grammar-based visualization tools like the Gosling toolkit could be converted into meaningful text descriptions, {% include get-name-degree.html title="Thomas Smits" %} led the AltGosling project to automate description generation for genome-mapped data visualizations.

> 

Reviewer: I don't really see the added value of adding degree titles to the text? It also makes it a bit harder to read. If someone is interested in degrees, they can always check out the individual pages. Perhaps we can even hyperlink these directly from this text?

We began making remediations to our website to better conform to WCAG 2.1 level AA. This included adding alt text to over 100 images and ensuring accessible navigation throughout the site.

{% include get-name-degree.html title="Sofia Rojas" %} and {% include get-name-degree.html title="Andrew Mar" %} joined the lab as Research Assistants, contributing to several of the team's projects. 
{% include get-name-degree.html title="Andrew Mar" %}'s contributions were instrumental in the paper *Explaining Unfamiliar Genomics Data Visualizations to a Blind Individual through Transitions*, which was accepted for presentation at the AccessViz workshop at IEEE VIS 2024. 

We continue to explore ways to make biomedical data visualizations accessible for people with a wideer range of disabilities. 
The preprint *Using OpenKeyNav to Enhance the Keyboard-Accessibility of Web-based Data Visualization Tools*, based on an open-source software library by {% include get-name-degree.html title="Lawrence Weru" %}, explores how to improve the keyboard accessibility of web-based data visualization tools for people with motor disabilities.

> The preprint was accepted for presentation at the AccessViz workshop at IEEE VIS 2024.

Reviewer: ```suggestion
The preprint was accepted for presentation at the AccessViz workshop at IEEE VIS 2024. 

We also looked into mental models of visualizations related to individuals who are blind, led by by {% include get-name-degree.html title="Thomas Smits" %}. The preprint for this was also accepted for presentation at the AccessViz workshop at IEEE VIS 2024.
```

The HIDIVE Lab accessibility research, led by {% include get-member-url.html title="Sehi L'Yi" %}, initially focused on improving screen reader accessibility for data visualizations. 
Recognizing that grammar-based visualization tools like the Gosling toolkit could be converted into meaningful text descriptions, {% include get-member-url.html title="Thomas Smits" %} led the AltGosling project to automate description generation for genome-mapped data visualizations.

Recognizing the under-representation of people with disabilities in the biomedical sciences,
we began quantifying the state accessibility of research journals and data portals, crucial for biomedical research, in a project led by {% include get-member-url.html title="Sehi L'Yi" %}.

We began making remediations to our website to better conform to WCAG 2.1 level AA. This included adding alt text to over 100 images and ensuring accessible navigation throughout the site.

{% include get-member-url.html title="Sofia Rojas" %} and {% include get-member-url.html title="Andrew Mar" %} joined the lab as Research Assistants, contributing to several of the team's projects. 

> {% include get-member-url.html title="Andrew Mar" %}'s contributions were instrumental in the paper *Explaining Unfamiliar Genomics Data Visualizations to a Blind Individual through Transitions*, which was accepted for presentation at the AccessViz workshop at IEEE VIS 2024. 

Reviewer: ```suggestion
{% include get-member-url.html title="Andrew Mar" %}'s contributions were instrumental in the paper *Explaining Unfamiliar Genomics Data Visualizations to a Blind Individual through Transitions*, which was accepted for presentation at the AccessViz workshop at IEEE VIS 2024. 

We also looked into mental models of visualizations related to individuals who are blind, led by by {% include get-name-degree.html title="Thomas Smits" %}. The preprint for this was also accepted for presentation at the AccessViz workshop at IEEE VIS 2024.
```

{% include get-name-degree.html title="Sofia Rojas" %} and {% include get-name-degree.html title="Andrew Mar" %} joined the lab as Research Assistants, contributing to several of the team's projects. 
{% include get-name-degree.html title="Andrew Mar" %}'s contributions were instrumental in the paper *Explaining Unfamiliar Genomics Data Visualizations to a Blind Individual through Transitions*, which was accepted for presentation at the AccessViz workshop at IEEE VIS 2024. 

We continue to explore ways to make biomedical data visualizations accessible for people with a wideer range of disabilities. 
The preprint *Using OpenKeyNav to Enhance the Keyboard-Accessibility of Web-based Data Visualization Tools*, based on an open-source software library by {% include get-name-degree.html title="Lawrence Weru" %}, explores how to improve the keyboard accessibility of web-based data visualization tools for people with motor disabilities.
The preprint was accepted for presentation at the AccessViz workshop at IEEE VIS 2024. 

> 

Reviewer 2: Sehi to add a paragraph about the K

---
# make sure to add page to _data/teams.yml

name: Our Accessibility Journey
team_id: accessibility
active: true
---

<!-- Every web interface should be accessible, but inaccessible web experiences are the norm. -->

<!-- When the HIDIVE lab began our accessibility research, we first focused on improving screen reader accessibility for data visualizations.  -->
The HIDIVE Lab accessibility research, led by {% include get-member-url.html title="Sehi L'Yi" %}, initially focused on improving screen reader accessibility for data visualizations. 
Recognizing that grammar-based visualization tools like the Gosling toolkit could be converted into meaningful text descriptions, {% include get-member-url.html title="Thomas Smits" %} led the AltGosling project to automate description generation for genome-mapped data visualizations.

Recognizing the under-representation of people with disabilities in the biomedical sciences,

> we began quantifying the state accessibility of research journals and data portals, crucial for biomedical research, in a project led by {% include get-member-url.html title="Sehi L'Yi" %}.

Reviewer: ```suggestion
we began quantifying the state accessibility of research journals and data portals, crucial for biomedical research, in a project led by Sehi L'Yi.
```

We began making remediations to our website to better conform to WCAG 2.1 level AA. This included adding alt text to over 100 images and ensuring accessible navigation throughout the site.

{% include get-member-url.html title="Sofia Rojas" %} and {% include get-member-url.html title="Andrew Mar" %} joined the lab as Research Assistants, contributing to several of the team's projects. 

> {% include get-member-url.html title="Andrew Mar" %}'s contributions were instrumental in the paper *Explaining Unfamiliar Genomics Data Visualizations to a Blind Individual through Transitions*, which was accepted for presentation at the AccessViz workshop at IEEE VIS 2024. 

Reviewer: ```suggestion
Andrew Mar's contributions were instrumental in the paper *Explaining Unfamiliar Genomics Data Visualizations to a Blind Individual through Transitions*, which was accepted for presentation at the AccessViz workshop at IEEE VIS 2024.
```

We also looked into mental models of visualizations related to individuals who are blind, led by by {% include get-name-degree.html title="Thomas Smits" %}. The preprint for this was also accepted for presentation at the AccessViz workshop at IEEE VIS 2024.

> 

Reviewer 2: ```suggestion

Our accessibility efforts led to more opportunities to expand our work. {% include get-member-url.html title="Sehi L'Yi" %} has recently received the NIH K99/R00 Pathway to Independence Award to continue work on accessibility. This project proposes to increase the diversity of the genomics workforce through accessible data and visualization. This includes conducting comprehensive evaluations of existing biomedical data resources and building tools and frameworks that can increase the accessibility of biomedical research and education. This grant will help {% include get-member-url.html title="Sehi L'Yi" %} to build an independent research group to work on this area in the future.

```

We continue to explore ways to make biomedical data visualizations accessible for people with a wideer range of disabilities. 
The preprint *Using OpenKeyNav to Enhance the Keyboard-Accessibility of Web-based Data Visualization Tools*, based on an open-source software library by {% include get-member-url.html title="Lawrence Weru" %}, explores how to improve the keyboard accessibility of web-based data visualization tools for people with motor disabilities.
The preprint was accepted for presentation at the AccessViz workshop at IEEE VIS 2024.

> 

Reviewer 2: ```suggestion

Our accessibility efforts led to more opportunities to expand our work. {% include get-member-url.html title="Sehi L'Yi" %} has recently received the NIH K99/R00 Pathway to Independence Award to continue work on accessibility. This project proposes to increase the diversity of the genomics workforce through accessible data and visualization. This includes conducting comprehensive evaluations of existing biomedical data resources and building tools and frameworks that can increase the accessibility of biomedical research and education. This grant will help {% include get-member-url.html title="Sehi L'Yi" %} to build an independent research group to work on this area in the future.

```

---
# make sure to add page to _data/teams.yml

name: Our Accessibility Journey
team_id: accessibility
active: true
---

<!-- Every web interface should be accessible, but inaccessible web experiences are the norm. -->

<!-- When the HIDIVE lab began our accessibility research, we first focused on improving screen reader accessibility for data visualizations.  -->

> The HIDIVE Lab accessibility research, led by {% include get-member-url.html title="Sehi L'Yi" %}, initially focused on improving screen reader accessibility for data visualizations. 

Reviewer: ```suggestion
The HIDIVE Lab accessibility research, led by {% include get-member-url.html title="Sehi L'Yi" %}, initially focused on improving accessibility for genome-mapped data visualizations. 
```

> Recognizing that grammar-based visualization tools like the Gosling toolkit could be converted into meaningful text descriptions, {% include get-member-url.html title="Thomas Smits" %} led the AltGosling project to automate description generation for genome-mapped data visualizations.

Reviewer: ```suggestion
Recognizing that grammar-based visualization tools like the Gosling toolkit could be converted into meaningful text descriptions, {% include get-member-url.html title="Thomas Smits" %} led the AltGosling project to automate description generation for genomics data visualizations.
```
